std_msgs/Header header
geometry_msgs/PoseStamped[] ptsvec
